main/views.py

```diff
@@ -199,3 +212,18 @@ def form_valid(self, form: BootProcessForm) -> HttpResponse:
         """
         asyncio.run(_boot_process("root", form.cleaned_data))
         return super().form_valid(form)
+
+
+@require_POST
+@csrf_exempt
+def deposit_message(request: HttpRequest) -> HttpResponse:
+    """Upload point for broadcast messages for display to end user.
+
+    Args:
+      request: the triggering request.
+
+    Returns:
+      A NO_CONTENT response.
+    """
+    MESSAGES.append(request.POST["message"])
+    return HttpResponse(status=HTTPStatus.NO_CONTENT)
```

Comment:
Why is `NO_CONTENT` returned here? Is it the same in general for a successful POST?

Reply:
NO_CONTENT is a way to signify success but that no the response contains no other data and that no action is expected on the part of the client. The practical differences between this and a 200 are pretty small so it's just a nicety. What a POST would return depends on the context, in the case of a user submitting a form it's usually a redirect, in the case of an API call it would likely return the json representation of the created object.